src/samples/audio/detect_test.c

```diff
@@ -269,8 +275,12 @@ static struct comp_dev *test_keyword_new(const struct comp_driver *drv,
 	if (!cd)
 		goto fail;
 
+#ifndef KWD_NN_DETECT_TEST
```

Comment:
#if !KWD_NN_DETECT_TEST (this one works better when you replace it with a kconfig, as disabled can show up both as undefined and as 0)